src/vector/platform/ElectronPlatform.js

```diff
@@ -385,4 +385,8 @@ export default class ElectronPlatform extends VectorBasePlatform {
     getEventIndexingManager(): BaseEventIndexManager | null {
         return this.eventIndexManager;
     }
+
+    async setLanguage(preferredLangs: string[]): Promise<void> {
+        return this._ipcCall('setLanguage', preferredLangs);
```

Comment:
As per react-sdk side, may be better to await on the promise ourselves and potentially error if it fails, since the layer above won't be awaiting on it.